build-scripts/components/runc/build.sh

```diff
@@ -3,5 +3,6 @@
 export INSTALL="${1}/bin"
 mkdir -p "${INSTALL}"
 
+export SHIM_CGO_ENABLED=1
 make BUILDTAGS="seccomp apparmor" EXTRA_LDFLAGS="-s -w"
```

Comment:
While we're here, can we also link `runc` statically?

```suggestion
make BUILDTAGS="seccomp apparmor" EXTRA_LDFLAGS="-s -w" static
```